[ompl] Can't find_package ompl

**Describe the bug**
Doing `find_package(ompl CONFIG REQUIRED)` results in an error

**Environment**
- OS: Windows 10.0.19042
- Compiler: MSVC 19.29.30040

**To Reproduce**
Steps to reproduce the behavior:
1. `./vcpkg install ompl:x64-windows`
2. Create CMakeLists.txt with only `find_package(ompl CONFIG REQUIRED)`
3. Run CMake
4. Error occurs

**Expected behavior**
No error should occur

**Failure logs** 
```
CMake Error at C:/vcpkg/installed/x64-windows/share/ompl/omplConfig.cmake:35 (find_dependency):
  Unknown CMake command "find_dependency".
Call Stack (most recent call first):
  C:/vcpkg/scripts/buildsystems/vcpkg.cmake:788 (_find_package)
  CMakeLists.txt:3 (find_package)
```

**Additional context**
Add any other context about the problem here.

--- a/ports/ompl/0002_Fix_config.patch
+++ b/ports/ompl/0002_Fix_config.patch
@@ -1,11 +1,12 @@
 ﻿diff --git a/omplConfig.cmake.in b/omplConfig.cmake.in
-index 465de25..ad373fa 100644
+index 465de25d..d1e91929 100644
 --- a/omplConfig.cmake.in
 +++ b/omplConfig.cmake.in
-@@ -18,8 +18,10 @@ set(OMPL_MAJOR_VERSION @PROJECT_VERSION_MAJOR@)
+@@ -18,8 +18,11 @@ set(OMPL_MAJOR_VERSION @PROJECT_VERSION_MAJOR@)
  set(OMPL_MINOR_VERSION @PROJECT_VERSION_MINOR@)
  set(OMPL_PATCH_VERSION @PROJECT_VERSION_PATCH@)
  
++include(CMakeFindDependencyMacro)
 +find_dependency(Boost 1.58 COMPONENTS serialization filesystem system program_options)
 +find_dependency(Eigen3)
  set_and_check(OMPL_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
@@ -14,7 +15,7 @@ index 465de25..ad373fa 100644
  foreach(_dir @FLANN_INCLUDE_DIRS@;@ODE_INCLUDE_DIRS@;@SPOT_INCLUDE_DIRS@;@TRIANGLE_INCLUDE_DIR@;@FCL_INCLUDE_DIRS@;@PQP_INCLUDE_DIR@;@ASSIMP_INCLUDE_DIRS@;@OPENGL_INCLUDE_DIR@)
      if(_dir)
          list(APPEND OMPL_INCLUDE_DIRS "${_dir}")
-@@ -29,7 +31,7 @@ list(REMOVE_DUPLICATES OMPL_INCLUDE_DIRS)
+@@ -29,7 +32,7 @@ list(REMOVE_DUPLICATES OMPL_INCLUDE_DIRS)
  set(OMPL_INCLUDE_DIRS "${OMPL_INCLUDE_DIRS}" CACHE STRING "Include path for OMPL and its dependencies")
  
  set_and_check(OMPL_LIBRARY_DIR @PACKAGE_LIB_INSTALL_DIR@)
@@ -23,7 +24,7 @@ index 465de25..ad373fa 100644
  foreach(_dir @FLANN_LIBRARY_DIRS@;@ODE_LIBRARY_DIRS@;@SPOT_LIBRARY_DIRS@;@FCL_LIBRARY_DIRS@;@ASSIMP_LIBRARY_DIRS@)
      if(_dir)
          list(APPEND OMPL_LIBRARY_DIRS "${_dir}")
-@@ -40,7 +42,7 @@ set(OMPL_LIBRARY_DIRS "${OMPL_LIBRARY_DIRS}" CACHE STRING "Library path for OMPL
+@@ -40,7 +43,7 @@ set(OMPL_LIBRARY_DIRS "${OMPL_LIBRARY_DIRS}" CACHE STRING "Library path for OMPL
  
  find_library(OMPL_LIBRARIES NAMES ompl.${OMPL_VERSION} ompl
      PATHS ${OMPL_LIBRARY_DIR} NO_DEFAULT_PATH)
@@ -32,7 +33,7 @@ index 465de25..ad373fa 100644
      if(_lib)
          list(APPEND OMPL_LIBRARIES "${_lib}")
      endif()
-@@ -61,3 +63,4 @@ endif()
+@@ -61,3 +64,4 @@ endif()
  
  include(FindPackageHandleStandardArgs)
  find_package_handle_standard_args(ompl DEFAULT_MSG OMPL_INCLUDE_DIRS OMPL_LIBRARY_DIRS OMPL_LIBRARIES)

--- a/ports/ompl/vcpkg.json
+++ b/ports/ompl/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "ompl",
   "version-string": "1.5.1",
-  "port-version": 3,
+  "port-version": 4,
   "description": "The Open Motion Planning Library, consists of many state-of-the-art sampling-based motion planning algorithms",
   "homepage": "https://ompl.kavrakilab.org/",
   "dependencies": [

--- a/ports/ompl/vcpkg.json
+++ b/ports/ompl/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "ompl",
-  "version-string": "1.5.1",
+  "version": "1.5.1",
   "port-version": 4,
   "description": "The Open Motion Planning Library, consists of many state-of-the-art sampling-based motion planning algorithms",
   "homepage": "https://ompl.kavrakilab.org/",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4862,7 +4862,7 @@
     },
     "ompl": {
       "baseline": "1.5.1",
-      "port-version": 3
+      "port-version": 4
     },
     "omplapp": {
       "baseline": "1.5.1",

--- a/versions/o-/ompl.json
+++ b/versions/o-/ompl.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "55fdebdea02caaa9252c3e036fb360fa0cd5f79b",
+      "version-string": "1.5.1",
+      "port-version": 4
+    },
+    {
       "git-tree": "4385de7645c202d99bb25420d049f15cb54d9ed7",
       "version-string": "1.5.1",
       "port-version": 3

--- a/versions/o-/ompl.json
+++ b/versions/o-/ompl.json
@@ -1,8 +1,8 @@
 {
   "versions": [
     {
-      "git-tree": "55fdebdea02caaa9252c3e036fb360fa0cd5f79b",
-      "version-string": "1.5.1",
+      "git-tree": "cde8f8a96e5fc3c5764ca85632efa1b828868e26",
+      "version": "1.5.1",
       "port-version": 4
     },
     {

